vscode-extension/src/prompts/workItemInitPrompt.template.md

```diff
@@ -0,0 +1,172 @@
+- **Remote** (Required): Git remote name (default: "origin")
+- **Artifact Paths** (Required): Location hint for workflow artifacts (default: "auto-derived" uses `.paw/work/<slug>/`)
+- **Additional Inputs** (Optional): Comma-separated list of extra input files/URLs, or "none"
+
+### 5. Call Language Model Tool to Generate Prompt Templates
+
+After creating WorkflowContext.md, invoke the registered language model tool to create all nine prompt templates:
+
+**Tool Call:**
+```
+paw_create_prompt_templates(
+  feature_slug: "<generated_feature_slug>",
+  workspace_path: "{{WORKSPACE_PATH}}"
+)
+```
+
+The tool must produce these files under `.paw/work/<feature-slug>/prompts/`:
+- 01A-spec.prompt.md
+- 01B-spec-research.prompt.md
+- 02A-code-research.prompt.md
+- 02B-impl-plan.prompt.md
+- 03A-implement.prompt.md
+- 03B-review.prompt.md
+- 04-docs.prompt.md
+- 05-pr.prompt.md
+- 0X-status.prompt.md
+
+Each prompt template must follow this exact format:
+
+```markdown
+---
+mode: <PAW-XX Agent Name>
+---
+
+<Instruction text> .paw/work/<feature-slug>/WorkflowContext.md
+```
+
+If the tool reports errors, surface the failure details to the user and stop the workflow.
+
+### 6. Git Branch Operations
+
+1. Check whether branch `{{TARGET_BRANCH}}` already exists locally with `git rev-parse --verify {{TARGET_BRANCH}}`
+   - If it exists, prompt the user to either checkout the existing branch or provide a different name
+2. Check for uncommitted changes with `git status --porcelain`
+   - If changes exist, warn the user and confirm they want to proceed before creating the branch
+3. Create and checkout the branch with `git checkout -b {{TARGET_BRANCH}}`
+   - Handle command failures gracefully and provide actionable guidance if the operation cannot complete
+
+### 7. Open WorkflowContext.md
+
+When all steps succeed, open `.paw/work/<feature-slug>/WorkflowContext.md` in the editor so the user can review the generated details immediately.
+
+## Error Handling Expectations
+
+- **Slug conflicts**: Offer the user options to append a numeric suffix automatically, choose an alternate slug, or cancel the workflow
+- **Existing branches**: Allow users to reuse the branch or choose a new name
+- **Uncommitted changes**: Warn users before continuing and respect their decision
+- **Git errors**: Provide clear error messages and suggested fixes (for example, commit changes or resolve conflicts)
+- **Network failures**: If the issue or work item cannot be fetched, fall back to branch-derived titles and inform the user
+- **Tool failures**: If `paw_create_prompt_templates` fails, show exact errors and do not continue to git operations
+
+## Success Confirmation
+
+Announce completion with a summary containing:
+✅ Created directory: `.paw/work/<feature-slug>/`
+✅ Generated WorkflowContext.md with all required fields
+✅ Successfully called `paw_create_prompt_templates`
+✅ Created nine prompt template files
```

Comment:
Justy say created prompt template files, to avoid having to change "nine" to another number if the prompt files change.